translations/de/module_de_lang.php

```diff
@@ -15,13 +15,22 @@
     'wd_account_holder_title' => 'Kontoinhaber',
     'wd_amount' => 'Betrag',
     'wd_bic' => 'BIC',
+    'wd_birthdate_format_php_code' => 'd.m.Y',
```

Comment:
I'd make this more generic and just call it `wd_date_format`.

Reply:
i changed it to wd_date_format_php_code, the php_code part should make it clear for the translator, that this format is needed for the plugin to work correctly